### ISSUE TITLE
[CAL-205] Sync issue when deleting appointments in Google Calendar

### ISSUE THREAD

**Issue by milospuac** [calcom/cal.com#5124]
![Screenshot 2022-10-20 at 10.31.10.png](https://uploads.linear.app/e86bf957-d82f-465e-b205-135559f4b623/786d0236-f06e-4110-b1bc-17a4e2d19312/bdc521bd-cf3e-4c22-b714-198b493482ec)

[https://www.loom.com/share/b4db8447c56b4ae1ac5ee6afd3cfef54](https://www.loom.com/share/b4db8447c56b4ae1ac5ee6afd3cfef54)

> From [Linear-GitHub Sync](https://synclinear.com)

**Comment by joeauyeung** [comment]
The Google calendar API can send push notifications to our app if an event changes.

[https://developers.google.com/calendar/api/guides/push#responding-to-notifications](https://developers.google.com/calendar/api/guides/push#responding-to-notifications)

**Comment by leog** [comment]
@zomars @emrysal the final decision here is that we don't really have control over what happens outside Cal.com platform, any change in the calendar outside the platform will not sync and therefore not taken into consideration for any calculations, am I right? just to close this ticket

**Comment by zomars** [comment]
That's right. Unless a contributor would like to step up and come up with some kind of webhook approach to keep things in sync. But the effort is it out of scope for core team for now.

**Comment by leog** [comment]
Thanks @zomars for the confirmation. Closing.

**Comment by emrysal** [comment]
Re-opening for discussion; Bi-directional sync is needed if we want to be able to authoritatively check for conflicts using upcoming bookings; because the slot that is moved in an external calendar will otherwise keep blocking a now "rescheduled" slot.

**Comment by albertborsos** [comment]
This would be a nice feature.

If I have opening hours from 9-17, and one of my customer books the 9-11 range, then asks to come at 8. In this case I cannot reschedule in the app, because this is not my offical opening hour range.

But I can easily drag the booking in google calendar. In this case the original cal.com booking still in the app, and blocks my free 10-11 slot.

**Comment by albertborsos** [comment]
Another use case which cause issues:

- client books an event
- client tells me the appointment time is not OK
- I open my calendar and delete the event (or my customer)
- in this case the event is still exists in cal.com:
    - the customer receives reminder email notification before the canceled event
    - the canceled event slot is not freed up int the cal.com booking UI, but it is free in my calendar

**Comment by marianlopezaz** [comment]
Hey everyone! an updates on this?

**Comment by copostic** [comment]
Hey guys! 
Any news on this issue?
Could we do something to help?

**Comment by max-mykhailenko** [comment]
Do you have this feature somewhere in the roadmap?

**Comment by CarinaWolli** [comment]
duplicate https://github.com/calcom/cal.com/issues/21053
